src/components/global/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styles from "./Dialog.module.scss";
+
+type TDialog = {
+  closeOnOutsideClick: boolean,
```

Comment:
on deriv.app we name the variables with snake_case, instead of camelCase used for functions. 
is it worth to apply this approach for this project?